phases/03-deep-learning-core/10-mini-framework/code/main.py

```
@@ -0,0 +1,603 @@
+    model = Sequential(
+        Linear(2, 16),
+        ReLU(),
+        Linear(16, 16),
+        ReLU(),
+        Linear(16, 8),
+        ReLU(),
+        Linear(8, 1),
+        Sigmoid(),
+    )
+
+    print(f"Model: 4 linear layers (2->16->16->8->1)")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)